SECURITY.md

```diff
@@ -0,0 +1,104 @@
+# Security
+
+Ensuring the security of the Splits Ecosystem is our utmost priority. We have invested a significant amount of time and money into 
+designing, testing, and auditing our contracts to maximize their safety and reliability. However, we recognize security is an ongoing process.
+If you believe you have found a security vulnerability, please read the following guidelines and report it to us.
+
+## Bug Bounty
+
+### Overview
+
+Starting on September 1, 2024, the [Splits contracts monorepo](https://github.com/0xSplits/splits-contracts-monorepo) is subject to
+the Splits Contracts Monorepo Bug Bounty (the "Program") to incentivize responsible bug disclosure.
+
+We are limiting the scope of the Program to [critical and high severity bugs](https://immunefi.com/immunefi-vulnerability-severity-classification-system-v2-3/), and are offering a reward of up to
+$25,000. Happy hunting!
+
+### Scope
+
+The scope of the Program is limited to bugs that result in the draining of user funds.
+
+The Program does NOT cover the following:
+
+- Code located in [test](./test) or [script](./script) directories.
+- External code in `node_modules`, except for code that is explicitly used by a deployed contract located in the
+  [src](./src) directory.
+- Contract deployments on test networks, such as Sepolia.
+- Bugs in third-party contracts or platforms interacting with our contracts.
+- Previously reported or discovered vulnerabilities in contracts built by third parties on our contracts.
+- Bugs that have already been reported.
+
+Vulnerabilities contingent upon the occurrence of any of the following also are outside the scope of this Program:
+
+- Front-end bugs (clickjacking etc.)
```

Comment:
ik from past bounties that people tend to submit bounties that require the network to fail or even bigger that requires use of a lot of compute to generate signatures. I guess `Compromise or misuse of third party systems or services` covers it in some sense but wonder if we want to make it more explicit.